spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/JpaAutoConfiguration.java

```diff
@@ -118,4 +119,6 @@ public DeadLetterQueueProviderConfigurerModule deadLetterQueueProviderConfigurer
                                                                         .build()
         );
     }
+
+    // end::JpaDeadLetterQueueProviderConfigurerModule[]
```

Comment:
Super duper mega nit:

```suggestion
    // end::JpaDeadLetterQueueProviderConfigurerModule[]
```